### ISSUE TITLE
build error MSB3721 CUDA 10.0

### ISSUE THREAD

**Issue by HamzahNizami** [AlexeyAB/darknet#1994]
Hi there, really need help. Keep getting the error below, when i try and build. I have followed your guide step by step but still cant get it built. 

CUDA: 10.0
OpenCV: 3.3
CuDNN: 7.4.1
MSVS: 2017 community edition

Any help would be appreciated been stuck on this for ages!

Severity	Code	Description	Project	File	Line	Suppression State
Error	MSB3721	The command ""C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\bin\nvcc.exe" -gencode=arch=compute_30,code=\"sm_30,compute_30\" -gencode=arch=compute_52,code=\"sm_52,compute_52\" --use-local-env -ccbin "C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.16.27023\bin\HostX86\x64" -x cu  -IC:\opencv\build\include -IC:\opencv\build\include\opencv2 -IC:\opencv\build\include\opencv -I..\..\3rdparty\include -I"C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\include" -I"C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\include" -I"C:\Program Files (x86)\NVIDIA GPU Computing Toolkit\cudnn\include" -I"C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\include"     --keep-dir x64\Release -maxrregcount=0  --machine 64 --compile -cudart static     -DOPENCV -DCUDNN -D_TIMESPEC_DEFINED -D_SCL_SECURE_NO_WARNINGS -D_CRT_SECURE_NO_WARNINGS -D_CRT_RAND_S -DGPU -DWIN32 -DNDEBUG -D_CONSOLE -D_LIB -D_MBCS -Xcompiler "/EHsc /W3 /nologo /O2 /Fdx64\Release\vc141.pdb /FS /Zi  /MD " -o x64\Release\network_kernels.cu.obj "C:\Users\H\Desktop\darknet-master\src\network_kernels.cu"" exited with code 1.	darknet	C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\IDE\VC\VCTargets\BuildCustomizations\CUDA 10.0.targets	712

**Comment by PeterQuinn925** [comment]
Start with stack overflow? https://stackoverflow.com/questions/22151402/how-can-i-resolve-the-error-the-command-exited-with-code-1

**Comment by Adesun** [comment]
just  open vs command prompt and run command just as above , see the message displayed.

**Comment by HamzahNizami** [comment]
> just open vs command prompt and run command as above , see the message displayed.

Sorry i dont understand what you mean. could you clarify?

**Comment by HamzahNizami** [comment]
> Start with stack overflow? https://stackoverflow.com/questions/22151402/how-can-i-resolve-the-error-the-command-exited-with-code-1

And i have tried this. Still unable to figure out the problem

**Comment by HamzahNizami** [comment]
> Start with stack overflow? https://stackoverflow.com/questions/22151402/how-can-i-resolve-the-error-the-command-exited-with-code-1

Thank you so much for this link. I was able to change my command prompt to give more details. After reading through the messages, i was able to figure out that it wasnt able to find a "specific path specified".

For anyone else having this problem, it was due to the fact that my CUDA folder had another folder called "v10.0" (may be a different version for others) and i didnt have it pointing to inside that folder. So to fix this is I just moved everything from inside the "v10.0" outside (into CUDA folder). Currently getting no errors and successful build.

Thanks a lot
